utility/portal/module.go

```diff
@@ -0,0 +1,75 @@
+package portal
```

Comment:
Let's remove the portal package for now because we added it to the spec after we started working on V1. 

I anticipate there will be a separate piece of work to add it in (the enum, this package, etc...) in the future, so I want to reduce the confusion.